src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -571,18 +767,18 @@ export default class OrgPerson extends Mixins(CommonMixin, OrgPersonMixin) {
       // populate mailing address
       // NB: add optional fields for later comparison
       this.inProgressMailingAddress = {
-        ...this.orgPerson.mailingAddress,
         deliveryInstructions: null,
-        streetAddressAdditional: ''
+        streetAddressAdditional: '',
+        ...this.orgPerson.mailingAddress
```

Comment:
This is the address bug fix that I previously committed to the release branch only. Now it will be in the main branch too.